scripts/languageserver/provider_completions.jl

```diff
@@ -8,7 +8,9 @@ function process(r::Request{Val{Symbol("textDocument/completion")},TextDocumentP
         s = get_sym(i)
         d = ""
         try 
```

Comment:
Actually, can we get rid of these ``try`` blocks? If there is an error, we should explicitly check for it and not rely on exception handling. And this kind of ``try`` block just hides bugs, imho.